sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormElement.cs

```diff
@@ -21,7 +21,7 @@ internal FormElement(BoundingBox boundingBox, int pageNumber, string text)
         /// images and inches for PDF. The <see cref="LengthUnit"/> type of a recognized page can be found
         /// at <see cref="FormPage.Unit"/>.
         /// </summary>
-        public BoundingBox BoundingBox { get; }
+        public FieldBoundingBox BoundingBox { get; }
```

Comment:
Just making sure that we didn't miss this by accident. The property name will be kept as `BoundingBox`, right?

Reply:
correct